identity-doctypes/src/commonMain/kotlin/com/android/identity/documenttype/knowntypes/UtopiaNaturalization.kt

```diff
@@ -19,7 +19,7 @@ object UtopiaNaturalization {
      */
     fun getDocumentType(): DocumentType {
         return DocumentType.Builder("Utopia Naturalization Certificate")
-            .addVcDocumentType(VCT)
+            .addVcDocumentType(vct = VCT, keyBound = true)
```

Comment:
In its original implementation, this document type was our non-keybound example. Are we changing that intentionally, now that we have the movie ticket to fill that use case?

And if we are changing this, are there any other changes required to switch this from keybound to non-keybound?

Reply:
I think Movie Ticket should be our example of a non-keybound credential. I think we should delete GermanPersonalID and UtopiaNaturalization (in a separate PR) since those were things we did in the context of the Funke.

Reply:
+1, deletion sounds good.